guides/v2.1/coding-standards/technical-guidelines.md

```diff
@@ -690,7 +690,7 @@ We are reviewing this section and will publish it soon.
 
 {% collapsible Example: %}
 ```php
-class SampleEventObserverThatModifiesInputs
+class SampleEventObserverThatModifiesInputs implements \Magento\Framework\Event\ObserverInterface
```

Comment:
@ronak2ram Thank you for reviewing.
We have updated the PR.
Can you, please, check it again?
Thank you!

Reply:
@serhiyzhovnir 
Thanks for the update. 
Please update little change.
Can you please revert changes of `use Magento\Framework\App\DataObject;`
Because this class not used in the code. it's just used in the code comment.

Reply:
@ronak2ram The changes were added.
Thank you!

Reply:
@serhiyzhovnir thanks for the update. PR looks good now.